Source/Core/DolphinQt/SkylanderPortal/SkylanderPortalWindow.cpp

```diff
@@ -344,7 +344,10 @@ QVBoxLayout* SkylanderPortalWindow::CreateFinderLayout()
   // translations in other languages, check the SuperChargers manual at
   // https://support.activision.com/manuals
   m_element_filter[8]->setText(tr("Undead"));
-  m_element_filter[9]->setText(tr("Other"));
+  m_element_filter[9]->setText(tr("Dark"));
+  m_element_filter[10]->setText(tr("Light"));
+  m_element_filter[11]->setText(tr("Kaos"));
```

Comment:
Please add the i18n comments for the new elements too. (You can skip it for Other.)

Reply:
The new elements have no Japanese names as far as I know, since only Spyro's Adventure was released in Japan.

Reply:
Wow. That was fast. Where did you find the Japanese translations? There doesn't seem to be a Japanese manual listed

Reply:
Okay. Done

Reply:
I had to search around for a while, but eventually I found them at http://blog.livedoor.jp/domo44/archives/1914472.html